code/jose/Python_Labs/lab02.py

```diff
@@ -5,8 +5,27 @@
 # hundreds_digit = x//100
 # print(hundreds_digit)
 
```

Comment:
[Lab 2: Number to Phrase](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/02%20Number%20To%20Phrase.md)

**Needs Work**

* Input of `104`:
    * Expecter output: `one hundred four`
    * Actual output:
    ```
    PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs> python .\lab02.py
    Enter a number: 94
    ninety four
    PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs> python .\lab02.py
    Enter a number: 104
    Traceback (most recent call last):
      File "C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs\lab02.py", line 82, in <module>
        print(hundreds_digit + 'hundred',tens_digitwritten[tens_digit],ones[ones_digit])
    KeyError: 0
    PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs>
    ```